actionpack/test/dispatch/routing_test.rb

```diff
@@ -4630,7 +4630,7 @@ def show
       get "/foo/show/%E2%EF%BF%BD%A6?something_else=%E2%EF%BF%BD%A6"
       assert_response :bad_request
 
-      get "/foo/show/%E2%EF%BF%BD%A6?something_else=%E2%EF%BF%BD%A6"
+      get "/bar/show/%E2%EF%BF%BD%A6?something_else=%E2%EF%BF%BD%A6"
```

Comment:
This is unrelated, but noticed that `"/bar/show(/:id)"` is currently not being used and thought maybe this was intended to use it..